src/Common/StatusInfo.cpp

```diff
@@ -2,9 +2,14 @@
 #include <Common/ExternalLoaderStatus.h>
 
 /// Available status. Add something here as you wish.
-#define APPLY_FOR_STATUS(M) \
+#define APPLY_FOR_BUILTIN_STATUS(M) \
     M(DictionaryStatus, "Dictionary Status.", DB::getStatusEnumAllPossibleValues()) \
 
+#ifdef APPLY_FOR_EXTERNAL_STATUS
```

Comment:
I see StatusInfo as unneeded - it was added for only one unimportant use-case; the code is not good (mostly copy-paste), and I'd like to remove it eventually.

Reply:
OK. I've reverted the changes in StatusInfo.